io.kapsa.drive.json

```diff
@@ -0,0 +1,338 @@
+{
+  "app-id": "io.kapsa.drive",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.vala"
```

Comment:
why do you need vala?

Reply:
If we don't specify that the: `ayatana-ido` build fails with:
```
CMake Error at /usr/share/cmake-3.26/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Vala (missing: VALA_COMPILER)
Call Stack (most recent call first):
  /usr/share/cmake-3.26/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindVala.cmake:52 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  src/CMakeLists.txt:164 (find_package)
```

I am not sure if there are better ways to overcome this issue.

Reply:
a CMake option to disable it. I don't know the package so I can't tell. Other have the option.

Reply:
apparently there is no choice.